sdk/src/main/java/io/radar/sdk/model/RadarUser.kt

```diff
@@ -250,9 +261,7 @@ class RadarUser(
         obj.putOpt(FIELD_SEGMENTS, RadarSegment.toJson(this.segments))
         obj.putOpt(FIELD_TOP_CHAINS, RadarChain.toJson(this.topChains))
         obj.putOpt(FIELD_SOURCE, Radar.stringForSource(this.source))
-        val fraudObj = JSONObject()
-        fraudObj.putOpt(FIELD_PROXY, this.proxy)
-        obj.putOpt(FIELD_FRAUD, fraudObj)
+        obj.putOpt(FIELD_FRAUD, this.fraud.toJson())
```

Comment:
Is this ok? In this case, FRAUD will always be included in the json, even if proxy and mocked are unknown/set to false.

Reply:
the use of the static methods for segments and chains is used for collections of segments or chains, so it is not needed. This is closer to the implementation for place, trip, country, etc.